src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -169,11 +179,49 @@ def generateBaseMesh(cls, region, options):
                         lNodeIds[n3][n2][n1] = nodeIdentifier
                         nodeIdentifier += 1
 
+            # Upper lobe nodes
+            for n3 in range(uelementsCount3 + 1):
+                reset = 0
+                uNodeIds.append([])
+                for n2 in range(uelementsCount2 + 1):
+                    uNodeIds[n3].append([])
+                    for n1 in range(uelementsCount1 + 1):
+                        uNodeIds[n3][n2].append(None)
+                        if (n1 == 0 or n1 == uelementsCount1) and (n2 == 0 or n2 == uelementsCount2):
+                            continue
+                        if (n2 < uelementsCount2 - 2) and (n3 < uelementsCount3 - 2):
+                            continue
+                        if (n2 == 0 or n2 == uelementsCount2) and (n3 == uelementsCount3):
+                            continue
+                        if (n1 == 0 or n1 == uelementsCount1) and (n3 == uelementsCount3):
+                            continue
+
+                        # Oblique fissure nodes
+                        if reset < 3 and n3 < uelementsCount3 - 2:
```

Comment:
I'm finding ``reset`` hard to understand. Can you change this logic to be purely in terms of n1, n2 and n3.
Also remember brackets.